docs/build-win.md

```diff
@@ -0,0 +1,292 @@
+# SRT Build Instructions
+
+<!-- TOC -->
+
+- [1. Prerequisites](#1-prerequisites)
+  - [1.1. Build Tool Dependencies](#11-build-tool-dependencies)
+  - [1.2. External Library Dependencies](#12-external-library-dependencies)
+    - [1.2.1. Cryptograpjic Library](#121-cryptograpjic-library)
+    - [1.2.2. Threading Library](#122-threading-library)
+  - [1.3. Package Managers](#13-package-managers)
+    - [1.3.1. VCpkg Packet Manager (optional)](#131-vcpkg-packet-manager-optional)
+    - [1.3.2. NuGet Manager (optional)](#132-nuget-manager-optional)
+- [2. Preparing Dependencies](#2-preparing-dependencies)
+  - [2.1. Cryptograpjic Library](#21-cryptograpjic-library)
+    - [2.1.1. Install OpenSSL](#211-install-openssl)
+      - [2.1.1.1. Using vcpkg](#2111-using-vcpkg)
+      - [2.1.1.2. Using Installer](#2112-using-installer)
+      - [2.1.1.3. Build from Sources](#2113-build-from-sources)
+    - [2.1.2. Install MbedTLS](#212-install-mbedtls)
+    - [2.1.3. Install LibreSSL](#213-install-libressl)
+  - [2.2. Threading Library](#22-threading-library)
+    - [2.2.1. Using C++11 Threading](#221-using-c11-threading)
+    - [2.2.2. Building PThreads](#222-building-pthreads)
+      - [2.2.2.1. Using vcpkg](#2221-using-vcpkg)
+      - [2.2.2.2. Using NuGet](#2222-using-nuget)
+      - [2.2.2.3. Build pthreads4w from Sources](#2114-build-pthreads4w-from-sources)
+      - [2.2.2.4. Build pthread-win32 from Sources](#2114-build-pthread-win32-from-sources)
+- [3. Building SRT](#3-building-srt)
+  - [3.1. Cloning the Source Code](#31-cloning-the-source-code)
+  - [3.2. Generate Build Files](#32-generating-build-files)
+  - [3.3. Build SRT](#33-build-srt)
+
+<!-- /TOC -->
+
+## 1. Prerequisites
+
+### 1.1. Build Tool Dependencies
+
+The following are the recommended prerequisites to build `srt` on Windows.
+
+- [CMake](http://www.cmake.org/) v2.8.12 or higher (cross-platform family of tools to build software)
+- [git](https://git-scm.com/about) client (source code management tool)
+- [Visual Studio](https://visualstudio.microsoft.com/vs/downloads/) (compiler and linker)
+
+### 1.2. External Library Dependencies
+
+#### 1.2.1. Cryptograpjic Library
+
+SRT has an external dependency on **cryptographic library**.
+This dependency can be disabled with the `-DENABLE_ENCRYPTION=OFF` CMake build option.
+Then SRT will be able to operate only in unencrypted mode.
```

Comment:
I'd write it rather this way.

This dependency can be disabled by `-DENABLE_ENCRYPTION=OFF` CMake build option. With disabled encryption SRT will be unable to make or accept encrypted connections.

Reply:
With the disabled `enforced encryption` SRT will be able to establish a connection.

Reply:
Replacing with the following text
>This dependency can be disabled by `-DENABLE_ENCRYPTION=OFF` CMake build option.
With disabled encryption SRT will be unable to estabish secure connections,
only unencrypted mode can be used.

Reply:
Good, just mind  that you made a typo again in "estabish".